source/includes/aws/_buckets.md

```diff
@@ -72,6 +73,42 @@ curl -X POST \
 | `taskId` <br/>*string*     | The task id related to the bucket creation. |
 | `taskStatus` <br/>*string* | The status of the operation.                  |
 
+
+<!-------------------- Update bucket -------------------->
+
+#### Update Bucket Permissions
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/rest/services/aws/test-env/buckets?operation=edit_bucket_privileges"
+```
+
+> Request body examples:
+
+```json
+{
+	"access": "public-read"
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+    "taskId": "30121175-926a-4fd2-991b-ff303ffdf905",
+    "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/buckets/:regionName/:bucketName?operation=edit_bucket_privileges</code>
+
+Update the canned ACL of a buckets from Amazon S3.
```

Comment:
canned?

Reply:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl

Reply:
Thanks, Is it common to S3 or only AWS?

Reply:
I believe it is common to S3 (Pure also use canned acl). However, Pure supports a subset of AWS (I think they only support private)